hpcgap/src/gvars.c

```diff
@@ -534,12 +533,12 @@ UInt GVarName (
            CopiesGVars[gvar_bucket] = NewGVarBucket();
            FopiesGVars[gvar_bucket] = NewGVarBucket();
         }
-        SET_ELM_PLIST(ValGVars[gvar_bucket], gvar_index, 0);
-        SET_ELM_PLIST(NameGVars[gvar_bucket], gvar_index, string);
-        SET_ELM_PLIST(WriteGVars[gvar_bucket], gvar_index, INTOBJ_INT(1));
-        SET_ELM_PLIST(ExprGVars[gvar_bucket], gvar_index, 0);
-        SET_ELM_PLIST(CopiesGVars[gvar_bucket], gvar_index, 0);
-        SET_ELM_PLIST(FopiesGVars[gvar_bucket], gvar_index, 0);
+        SET_ELM_GVAR_LIST(ValGVars, gvar, 0);
```

Comment:
This is crashing because `gvar_index` refers to the location of `CountGVars`, not `gvar`